### PR TITLE
fandom-cleanup: add new template for Wikia sites

### PR DESCRIPTION
**Pull request by JohnyP36**
I'm working on making a template for issue #490

### CODE REVIEW

File: data/filters/templates/fandom-bloatware.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  - name: search
+    description: Hides "Top Trending Searches" when you open search
+    type: checkbox
+    default: true
+  - name: edit-buttons
+    description: Hide edit buttons at the top-left of every page/article
+    type: checkbox
+    default: true
+tags:
+  - fandom
```
Comment (JohnyP36) [inline data/filters/templates/fandom-bloatware.yaml:42]: I added the tag `fandom` but it has currently not its own page. So, that needs to be create too.

File: data/filters/templates/fandom-bloatware.yaml [hunk 2]
```diff
@@ -0,0 +1,126 @@
+title: "Fandom: interface cleanups"
```
Comment (JohnyP36) [inline data/filters/templates/fandom-bloatware.yaml:1]: If some of the @letsblockit/contributors has a better title, let me know

File: data/filters/templates/fandom-bloatware.yaml (hunk 1 above)
Comment (JohnyP36) [reply data/filters/templates/fandom-bloatware.yaml:42]: I came to the conclusion that this is not necessary.

File: data/filters/templates/fandom-cleanup.yaml [hunk 3]
```diff
@@ -0,0 +1,127 @@
+title: "Fandom: Wiki pages interface cleanups"
+contributors:
+  - JohnyP36
+  - BPower0036
```
Comment (xvello) [inline data/filters/templates/fandom-cleanup.yaml:4]:
```suggestion
contributors:
  - BPower0036
  - JohnyP36
  - lightbulbmeow
```

Let's credit @lightbulbmeow as they came up with the rules. Also, let's keep them in alphabetical order

**Review by xvello — APPROVED** [review]
Looks good to me! I didn't test the rules because there's already been two pair of eyes on them.

I rebased and added @lightbulbmeow to the contributors, feel free to press merge when you think you're done!

**Comment by JohnyP36** [comment]
> feel free to press merge when you think you're done!

Thank you for merging. Due to family related "issues" (not bad ones btw) I was not online yesterday.

**Pull request merged**